Common Collections/Hashmaps/Fruits Delivery/task-info.yaml

```diff
@@ -3,7 +3,7 @@ files:
 - name: src/main.rs
```

Comment:
Maybe we could place lib.rs before main.rs here in yaml, so that when the student opens this task lib.rs would be shown in the code editor instead of main.rs